humbug.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Note that you don`t share phpunit.xml int your projects. You would rather share phpunit.xml.dist. 
For example my humbug.json in humbug often changes its contents. If it is in repository there is a chance that anyone can commit by accident with invalid contents.